pkg/pbflow/proto.go

```diff
@@ -247,3 +251,19 @@ func macToUint8(mac uint64) [6]uint8 {
 		uint8(mac),
 	}
 }
+
```

Comment:
another UT is needed for this

Reply:
done in https://github.com/netobserv/netobserv-ebpf-agent/pull/820/commits/a4849cb1d206b1fea3ed73c72114f96f25d354b2